tools/rapids-retry

```diff
@@ -26,25 +29,63 @@
 #                          seconds, to wait between retries.
 #                          Default is a 10 second sleep
 #
+# Additional options:
+#
+# --quiet                - when specified as the first argument after the command,
+#                          captures the command's output instead of sending it to stdout.
+#                          Logger messages are redirected to stderr.
+#                          Returns the output of the command on success.
+#
```

Comment:
As per the implementation, `--quiet` needs to be necessarily the first argument if specified. I would either document that or make parsing more robust, I would prefer the latter but I do realize a lot of us are time-constrained so just documenting would suffice, something like this would suffice:

```
$0 [--quiet] <command>
```